src/mysql_vm_helpers.py

```diff
@@ -331,6 +331,29 @@ def reconfigure_mysqld(self) -> None:
         logger.debug("Installing charmed-mysql snap")
         charmed_mysql.ensure(snap.SnapState.Latest, channel="8.0/edge")
 
+    @staticmethod
+    def write_content_to_file(
```

Comment:
should we create an abstract method in the `mysql.py` library for this function?

Reply:
not yet - but will eventually. It will require a bit more thought, as in a dedicated PR